src/instructlab/model/download.py

```diff
@@ -48,38 +283,33 @@
 @click.pass_context
 @clickext.display_params
 def download(ctx, repository, release, filename, model_dir, hf_token):
-    """Download the model(s) to train"""
-    click.echo(f"Downloading model from {repository}@{release} to {model_dir}...")
-    if hf_token == "" and "instructlab" not in repository:
-        raise ValueError(
-            """HF_TOKEN var needs to be set in your environment to download HF Model.
-            Alternatively, the token can be passed with --hf-token flag.
-            The HF Token is used to authenticate your identity to the Hugging Face Hub."""
+    downloader = None
+
+    if is_oci_repo(repository):
+        downloader = OCIDownloader(
+            repository=repository, release=release, download_dest=model_dir
+        )
+    elif is_huggingface_repo(repository):
+        downloader = HFDownloader(
+            repository=repository,
+            release=release,
+            download_dest=model_dir,
+            filename=filename,
+            hf_token=hf_token,
+            ctx=ctx,
+        )
+    else:
+        click.secho(
+            "repository supplied matches neither Huggingface, nor OCI registry format. Please supply a valid repository",
```

Comment:
Please print `repository` when this message is shown.

Reply:
fixed, thanks